### PR TITLE
[TASK] Avoid warnings on PHP 8 in examples

### PR DESCRIPTION
**Pull request by brotkrueml**
Additionally, use ??= operator as TYPO3 v11 uses PHP 7.4+,
which simplifies the code.

### CODE REVIEW

File: Documentation/ApiOverview/CachingFramework/Configuration/Index.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ the cache system falls back to the default backend and default frontend settings
 
 .. code-block:: php
 
-   if (!is_array($GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['myext_mycache'])) {
+   if (!is_array($GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['myext_mycache'] ?? false)) {
```
Comment (mbrodala) [inline Documentation/ApiOverview/CachingFramework/Configuration/Index.rst:33]: Why not use `??=` here too?

Comment (brotkrueml) [reply Documentation/ApiOverview/CachingFramework/Configuration/Index.rst:33]: I adapted the recent version: What, if the key myext_mycache is not an array but a string or a bool? Strange, but that is the check ;-)

Comment (mbrodala) [reply Documentation/ApiOverview/CachingFramework/Configuration/Index.rst:33]: Indeed, I missed the `is_array()` here. Still I don't think it's the responsibility of this code to fix a broken base IMO.

File: Documentation/ApiOverview/CachingFramework/Configuration/Index.rst [hunk 2]
```diff
@@ -30,7 +30,7 @@ the cache system falls back to the default backend and default frontend settings
...
        $GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['myext_mycache'] = [];
    }
```
Comment (mbrodala) [inline Documentation/ApiOverview/CachingFramework/Configuration/Index.rst:35]:
```suggestion
   $GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['myext_mycache'] ??= [];
```

**Review by linawolf — APPROVED** [review]

**Pull request merged**